### DISCUSSION TITLE
🙋🙋 About  adaptation to mobile browsers / 关于手机浏览器的适配

### DISCUSSION THREAD

**Discussion by sheepzh** [sheepzh/time-tracker-4-browser#300] [Ideas]
I have received some optimization features for mobile browsers. But I don't know which mobile browsers you use most often.

So if you are using the extension on your mobile browser, feel free to tell me what browser you use and how to install the extension.

Thank you guys!

--------------------

我之前经常有收到一些，针对手机浏览器的优化请求。但是我不太知道你们平时在哪些手机浏览器使用这个扩展。

所以如果你正在移动浏览器上使用该扩展程序的话，请随时留言浏览器的名称、版本，及其扩展的安装方式。

非常感谢大家！！！

**Comment by herrbrixter** [comment]
I'm using Firefox Android browser. I hope your Time Tracker could support Firefox mobile browser too. Thanks!

**Reply by sheepzh** [reply]
> replies

@herrbrixter Thanks for your feedback, the progress is tracked in issue #394

**Comment by pablopovar** [comment]
Firefox Android @sheepzh
